### PR TITLE
more-less: fix issue with aria-hidden

### PR DESCRIPTION
**Pull request by margaree**
Issue: Can't have the more/less button be focusable but also have `aria-hidden` because that could cause confusion and fails axe tests.

I have removed `aria-hidden` and as an alternative added `aria-expanded` and `aria-controls`. I don't know that this is the ideal approach yet and am looking for some feedback (especially if others have a better understanding of `aria-expanded` and `aria-controls`).
- I _think_ `aria-controls` should actually be on the button, but that fails axe tests
- `aria-expanded` is a bit strange here. My impression is that `aria-expanded` communicates that if a person were to interact with that button then there would be hidden content (corresponding to `aria-controls`) that would appear. However, in this case the content isn't actually hidden to the voiceover user. One thing I think that could work here (which may be too hacky) is have `aria-expanded` always set to true, since the content is technically always expanded for voiceover users.

### CODE REVIEW

**Comment by dlockhart** [comment]
Previously with `aria-hidden`, did it actually hide the button to screen readers? Just trying to figure out if we had the "best" accessibility experience and we're just hacking around to make aXe pass, or if there's room for improvement here.

**Comment by svanherk** [comment]
> Previously with aria-hidden, did it actually hide the button to screen readers? Just trying to figure out if we had the "best" accessibility experience and we're just hacking around to make aXe pass, or if there's room for improvement here.

IIRC, it wasn't doing a good job of hiding it - I think you could still tab to it, and it would say something false (like "hidden" or "disabled" or "blank"), and you could click it, with no explanation to the user of what they'd done.

**Comment by margaree** [comment]
Yup so with voiceover on Safari, I can still tab to it (with `aria-hidden`) and it just says "group"

**Comment by dlockhart** [comment]
[I found this](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/hidden/), which likely explains it:

>>> You must never use aria-hidden="true" on any focusable element, or on any element that itself contains focusable children.

This is due to the fact that aria-hidden indeed is respected by screen readers even on focusable elements while reading them in browse mode - but during usage of focus mode, focusable elements are still reachable by the browser (with the Tab key), which leads to strange (or missing) announcements of hidden elements.

**Comment by dlockhart** [comment]
IMO, what you have here is probably the best of all the evils. I also like the option of just always treating it as "expanded" (which it is), but I'm a bit worried folks will try to collapse things and will be confused when it doesn't.

**Comment by margaree** [comment]
Okay cool - yeah it seemed like there weren't any great options here. Agreed about the potential confusion with always expanded.

File: components/more-less/more-less.js [hunk 1]
```diff
@@ -120,9 +121,11 @@ class MoreLess extends LocalizeMixin(LitElement)  {
 		this.__blurBackground = 'linear-gradient(rgba(255, 255, 255, 0) 0%, rgb(255, 255, 255) 100%)';
 		this.__transitionAdded = false;
 
+		this.expanded = false;
```
Comment (dlockhart) [inline components/more-less/more-less.js]: Should something toggle this to `true` when "more" is shown?

Comment (margaree) [reply components/more-less/more-less.js]: Yup that happens in here https://github.com/BrightspaceUI/core/blob/master/components/more-less/more-less.js#L282

Comment (dlockhart) [reply components/more-less/more-less.js]: Oh sorry I missed that it wasn't a new property!

**Pull request merged**